README.md

```diff
@@ -1,56 +1,49 @@
----
-title: Gatsby Code Profile 
----
-# Gatsby Code Profile
+# Gatsby Visual Studio Code Profile
 
-This is a custom code profile built for [GatsbyJS](gatsbyjs.org) code Demos, it makes us a Gatsby color scheme and a variation of [Shades of Purple](https://marketplace.visualstudio.com/items?itemName=ahmadawais.shades-of-purple) created by [Ahmad Awais](https://marketplace.visualstudio.com/publishers/ahmadawais)
+This is a custom VS Code profile built for [Gatsby](gatsbyjs.org) code demos and docs, providing an accessible Gatsby color scheme. 
 
-# What is a code profile ? 
-A code profile is essentially a custom `settings.json` configuration that is shareable  and resuseable across different instances of VScode. It is mostly used to avoid repetition that comes with resetting the default `settings.json` of a VScode environment when you want to say, bump up the font-size for a screencast or feel like using a theme with other custom settings.  
+This theme is a variation of [Shades of Purple](https://marketplace.visualstudio.com/items?itemName=ahmadawais.shades-of-purple) created by [Ahmad Awais](https://marketplace.visualstudio.com/publishers/ahmadawais).
 
-> You can also customize which extensions load per code profile. 
+## What is a code profile? 
 
-# Setting up Gatsby_code_profiles
+A code profile is a custom `settings.json` configuration that's shareable and reusable across different instances of VS Code. It is mostly used to avoid repetition that comes with resetting the default `settings.json` of a VS Code environment when you want to say, bump up the font-size for a screencast or feel like using a theme with other custom settings. 
 
-To set up the Gatbsy code profile you would have to follow these steps:
+For the Gatsby docs, a VS Code profile helps us create consistent learning materials.
 
- 1. Clone repository into your root directory .
- ```shell 
+## Setting up Gatsby_code_profiles
+
+To set up the Gatsby code profile, follow these steps:
+
+ 1. Clone the repository **into your computer's root directory**.
+
+ ```shell
+ cd ~/
  git clone https://github.com/Ekwuno/gatsby_code_profiles.git
  ```
- Then change directory into the code profile.
+ Then change directories into the code profile.
  ```shell 
- git clone cd gatsby_code_profiles 
+ cd gatsby_code_profiles 
  ```
- In gatsby_code_profiles directory exists the presentation folder which houses the custom settings. 
- 
- > Multiple sub folders can exist for code profiles and you would have to change directory to access the setting.
+ In the `gatsby_code_profiles` directory exists the presentation folder which houses the custom settings.  
  
- 
- 2. Set up alias
-
-Aliases are helpful for shortening commands and also help to reduce the cognitive load of learning a really long command and having to type it everytime. Vscode has a command line alias to [launch from the command line](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line) and this comes 
```

Comment:
This section seemed like an unnecessary step for setting up the profile, let me know what you think though.

Reply:
I think it's necessary to emphasize the need for aliases, Maybe I should make that an external link? Also, adding the link to the VScode Lunch command is to show how to set that up and what it is about, as the `code` command becomes part of the `gatsby-tech` alias later on.